Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_043.NuGetCredentials.ps1

```diff
@@ -0,0 +1,36 @@
+<packageSources>
```

Comment:
Update the usernames and passwords so they can be more easily identified in the sarif file.